src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1549299689.inc.php

```diff
@@ -6,26 +6,38 @@
-communication        # Communication         # Kommunikation         # fas fa-at
-dataexchange         # Data exchange         # Datenaustausch        # fas fa-exchange-alt
-layout               # Layout                # Layout                # far fa-address-card
-internationalization # Internationalization  # Internationalisierung # fas fa-flag
-internalusers        # Internal users        # Interne Benutzer      # fas fa-user-lock
-logs                 # Logs                  # Logs                  # fas fa-clipboard-list
-routing              # Routing               # Routing               # fas fa-random
-system               # System                # System                # fas fa-cog
+base # contents             # Contents              # Inhalte               # fas fa-file-alt
+shop # products             # Products & categories # Produkte & Kategorien # fas fa-cubes
+shop # productlists         # Product lists         # Produktlisten         # fas fa-boxes
+shop # discounts            # Discounts & vouchers  # Rabatte & Gutscheine  # fas fa-gift
+shop # orders               # Orders                # Bestellungen          # fas fa-shopping-basket
+shop # checkout             # Checkout              # Checkout              # fas fa-truck-monster
+base # externalusers        # Customers / users     # Kunden / Benutzer     # fas fa-user
+shop # ratings              # Shop ratings          # Shop-Bewertungen      # fas fa-star
+base # search               # Search                # Suche                 # fas fa-search
+base # analytics            # Analytics             # Analyse               # fas fa-chart-pie
+base # communication        # Communication         # Kommunikation         # fas fa-at
+base # dataexchange         # Data exchange         # Datenaustausch        # fas fa-exchange-alt
+base # layout               # Layout                # Layout                # far fa-address-card
+base # internationalization # Internationalization  # Internationalisierung # fas fa-flag
+base # internalusers        # Internal users        # Interne Benutzer      # fas fa-user-lock
+base # logs                 # Logs                  # Logs                  # fas fa-clipboard-list
+base # routing              # Routing               # Routing               # fas fa-random
+base # system               # System                # System                # fas fa-cog
 EOT;
 
 $categoryLines = \explode(PHP_EOL, $categoryDef);
 $position = 0;
 
+$isShopSystem = \class_exists(TShop::class);
+
 foreach ($categoryLines as $categoryLine) {
     $id = TCMSLogChange::createUnusedRecordId('cms_menu_category');
-    [$systemName, $nameEn, $nameDe, $iconFontCssClass] = \preg_split('/\s+#\s+/', $categoryLine);
+    [$systemType, $systemName, $nameEn, $nameDe, $iconFontCssClass] = \preg_split('/\s+#\s+/', $categoryLine);
+
+    if (false === $isShopSystem && 'shop' === $systemType) {
```

Comment:
It is no shop system but the system type is "shop"?

Maybe name it entryType?

Reply:
It is a type that refers to what kind of "system" is run. So I'd say the name fits - we check if the current system is a shop against the type of system that is required by the entry.